components/google-cloud/container/gcp/Dockerfile

```diff
@@ -17,6 +17,6 @@ FROM python:3.7-slim
 RUN pip install -U google-cloud-aiplatform
 RUN pip install -U google-cloud-storage
 
-COPY launcher .
+COPY google_cloud_pipeline_components/experimental/gcp_launcher .
```

Comment:
instead of directly copying the gcp_launcher I would suggest installing and loading the module from google_cloud_pipeline_components

Reply:
Agreed. I plan to do the changes in two steps - moving the code around, and merge the dockerfiles to a single one. i could make this change as part of the second PR.

Reply:
sgtm